components/menu-items/item.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import Shortcut from './shortcut';
+import IconButton from '../icon-button';
+import './style.scss';
+
+/**
+ * Renders the more menu item.
+ *
+ * @return {WPElement} More menu item.
+ */
+function MenuItemsItem( { className, icon, isSelected = false, label, onClick, shortcut } ) {
```

Comment:
Do you have a convention of putting optional arguments as the last one in a list of arguments?

Reply:
There is no pattern really, I put them sorted by name, can move `isSelected` 👍